README.md

```diff
@@ -59,6 +60,18 @@ $ tfplugindocs validate --help
 Usage: tfplugindocs validate [<args>]
 ```
 
+`migrate` command:
+
+```shell
+$ tfplugindocs migrate --help
+
+Usage: tfplugindocs migrate [<args>]
+
+    --examples-dir <ARG>             examples directory based on provider-dir                                                                                           (default: "examples")
+    --provider-dir <ARG>             relative or absolute path to the root provider code directory when running the command outside the root provider code directory
+    --templates-dir <ARG>            new website templates directory based on provider-dir; files will be migrated to this directory                                    (default: "templates")
```

Comment:
I think the name should be the same to keep it in parity with the `generate` command with the same flag. If we changed the flag to something like `--output-dir` it might not be obvious to the user that they would need to specify `generate --template-dir` to that directory after migration.

Reply:
Ah okay that's helpful context! Makes sense to me 👍🏻